builder/src/version.rs

```diff
@@ -0,0 +1,15 @@
+// Licensed under the Apache-2.0 license
+
+pub const ROM_VERSION_MAJOR: u16 = 1;
+pub const ROM_VERSION_MINOR: u16 = 0;
+pub const ROM_VERSION_PATCH: u16 = 1;
+
+// ROM Version - 16 bits
```

Comment:
We mentioned the version source could be changed down the road, but just wanted to call out we won't want to change the bit allocation for these 16 bits once we merge. So we should at least make sure we are all in agreement on that.

Reply:
It sounds like everyone is agreed this is a good format